virttest/utils_test/__init__.py

```diff
@@ -1514,7 +1513,7 @@ def service_setup(vm, session, directory):
         src = os.path.join(directory, rh_perf_envsetup_script)
         vm.copy_files_to(src, "/tmp/rh_perf_envsetup.sh")
         logging.info("setup perf environment for host")
-        commands.getoutput("bash %s host %s" % (src, rebooted))
+        process.run("bash %s host %s" % (src, rebooted), shell=True)
```

Comment:
Suggest `process.system`

Reply:
@luckyh Sorry, I'm not quite sure why process.system is better than process.run here. I am very glad if I can get more infomation